tests/integration/clients/test_postgresql.py

```diff
@@ -55,7 +67,8 @@ def setup_class(cls):
                 test_bit_array BIT(8)[],
                 test_not_null_array INTEGER[] NOT NULL,
                 test_int_array_2d INTEGER[][],
-                test_text_array_3d TEXT[][][]
+                test_text_array_3d TEXT[][][],
+                test_enum_mood test_enum_type_mood
```

Comment:
WDYT about adding a secnod col that's `test_enum_type_mood[]` to validate enum arrays?

Reply:
I tried and it fails as-is.  With that column type `data_type` is `ARRAY` and  `udt_name` is `_test_enum_type_mood` interestingly.

I'd like to keep addressing arrays out of scope of this PR if that's okay with you.

Reply:
Ticket to track supporting that: https://github.com/recap-build/recap/issues/416